builder/examples/basic/alicloud.json

```diff
@@ -13,7 +13,8 @@
     "ssh_username":"root",
     "instance_type":"ecs.n1.tiny",
     "internet_charge_type":"PayByTraffic",
-    "io_optimized":"true"
+    "io_optimized":"true",
+    "associate_public_ip_address":"true"
```

Comment:
Out of curiosity since the code modifies the private IP address path, why do we change this in the basic example?

Reply:
Basic example not set associate_public_ip_address or ssh_private_ip, it not work since this PR：https://github.com/hashicorp/packer-plugin-alicloud/pull/52/commits